### DISCUSSION TITLE
How to use Baidu map tile

### DISCUSSION THREAD

**Discussion by guoyao132** [iTowns/itowns#1630] [Q&A]
const tmsSource = new itowns.TMSSource({
      crs: 'EPSG:3857',
      format: 'image/png',
      url: this.mapURL,
      tileMatrixSet: 'PM',
      zoom: this.mapZoom,
    });
    const colorlayer = new itowns.ColorLayer('OPENSM', {
      source: tmsSource,
    })
    view.addLayer(colorlayer)

I can use Gaode map tiles normally. How can I use Baidu tiles

**Comment by gchoqueux** [comment]
Have you web service specification, on Baidu tiles?
Have you a url on the Baidu tiles web service?
